src/app/static/src/app/store/baseline/actions.ts

```diff
@@ -111,18 +111,18 @@ export const actions: ActionTree<BaselineState, DataExplorationState> & Baseline
                 .then((response) => {
                     if (response) {
                         const metadata = response.data;
-                        const avaliableResources = rootGetters ? rootGetters["baseline/selectedDatasetAvailableResources"] : {}
+                        const availableResource = rootGetters ? rootGetters["baseline/selectedDatasetAvailableResources"] : {}
                         const exceptions = { // where DatasetResource keys do not match ADRSchemas keys
                             pop: "population",
                             program: "programme"
                         }
                         const resources: { [k in keyof DatasetResourceSet]?: DatasetResource | null } = {}
 
-                        Object.entries(avaliableResources).forEach(([key, value]) => {
+                        Object.entries(availableResource).forEach(([key, value]) => {
                             // If an available resource has a value, find the resource (using the alternate schema key where needed)
                             // and add it to the committed object under the original key
                             const schemaKey = key in exceptions ? exceptions[key as "pop" || "program"] : key
-                            resources[key as keyof typeof resources] = value ? findResource(metadata, schemas[schemaKey as keyof ADRSchemas]) : null
+                            resources[key as keyof typeof resources] = key ? findResource(metadata, schemas[schemaKey as keyof ADRSchemas]) : null
```

Comment:
Actually I guess we could know if we don't have a resource because it didn't exist when we last fetched, or because we didn't have permissions to fetch it - since we could compare metadata with actual resource data. But we still don't know if we'll be able to pull it. 

But we could identify genuine new resources that way, and flag those (though we might not be able to pull them), and distinguish them from resources we don't have, but did have metadata for before (which we could assume we don't have perms to see - although those perms may have changed in the interim - though that would be less common I guess).

Reply:
We should have a quick chat regarding this approach.

Reply:
OK, as discussed, let's go with the current approach, so just the small variable rename suggestion outstanding.